### DISCUSSION TITLE
Question about new and legacy .apk build

### DISCUSSION THREAD

**Discussion by Mintonist** [airsdk/Adobe-Runtime-Support#1603] [General]
I use latest AIR 33.1.1.743
Is it normal difference for "res" folder in .apk file for one project that was builded as legacy and new?
Total size and structure of all directories inside .apk is so different. It confused me)

![image](https://user-images.githubusercontent.com/9267850/151703615-ecac6cac-486d-401d-af54-d2463e918667.png)

**Comment by FliplineStudios** [comment]
I'm seeing the same, with the new packaging I no longer have any directories for drawable, mipmap, xml, etc. and only have a variety of loose renamed xml files and renamed pngs.  No idea if that's normal or how devices will interpret them...

**Comment by ajwfrost** [comment]
Seems to be normal .. you get the same if you create a new Android project and export a release APK...
